.swiftlint.yml

```diff
@@ -16,6 +16,9 @@ opt_in_rules:
   - vertical_parameter_alignment
   - vertical_parameter_alignment_on_call
 
+disabled_rules:
+  - orphaned_doc_comment
```

Comment:
do we get false positives often enough that it's worth disabling at the project level rather than on a case-by-case basis?

Reply:
We get a fair amount of `internal` declarations (though that [should get fixed](https://github.com/realm/SwiftLint/issues/4573#issuecomment-1322627749), but we also have some blocks of text like:
```swift
/**
Some explanation about a method
*/

let some = code()
```

I was disabling them individually, but after a while I realized that more docs are always better so not sure we get any benefit from the rule.

Reply:
I'll get this merged and we can re-enable the rule when they fix that regression.

Reply:
👍